roles/test_operator/tasks/tempest-tests.yml

```diff
@@ -16,8 +16,8 @@
 #
 - name: Configuring tests to be executed via skiplist
   when: >
-   cifmw_test_operator_tempest_include_list is not defined or
-   cifmw_test_operator_tempest_tests_include_override_scenario | bool
+   _test_operator_tempest_include_list is not defined or
+   _test_operator_tempest_tests_include_override_scenario | bool
   block:
     - name: Copy list_allowed to artifacts dir
       ansible.builtin.copy:
```

Comment:
This would require running the entire main as part of the stages loop, I am generally aginst it in the scope of this patch having that the goal here is to allow running the controllers CR with different parameters based on the stage. I think maybe it can be a future task but for now I don't think it is required for what this patch is trying to achieve. So :-1: for me

Reply:
Ok, makes sense. Already discussed in other comment.